### ISSUE TITLE
Benchmark 2D SOM with COSMOS 10-year catalog

### ISSUE THREAD

**Issue by dkirkby** [LSSTDESC/pz_bayes#3]
This issue is to take a first look and see if our simple training loop is efficient enough to get results in < 10 mins, and explore the hyperparams (grid size, learning rate, etc).
